recipes-support/ostree-pending-reboot/ostree-pending-reboot_0.1.bb

```diff
@@ -0,0 +1,23 @@
+SUMMARY = "OStree Pending Reboot service"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+inherit allarch systemd
+
+RDEPENDS_${PN} = "aktualizr-get" # TODO: ??
```

Comment:
The service itself does not require aktualizr, it just acts due an action caused by it.